src/Umbraco.Core/Constants-SqlTemplates.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Umbraco.Core
+{
+    public static partial class Constants
+    {
+        public static class SqlTemplates
+        {
+            public const string VersionableRepositoryGetVersionIds = "Umbraco.Core.VersionableRepository.GetVersionIds";
+            public const string VersionableRepositoryGetVersion = "Umbraco.Core.VersionableRepository.GetVersion";
+            public const string VersionableRepositoryGetVersions = "Umbraco.Core.VersionableRepository.GetVersions";
+            public const string VersionableRepositoryEnsureUniqueNodeName = "Umbraco.Core.VersionableRepository.EnsureUniqueNodeName";
+            public const string VersionableRepositoryGetSortOrder = "Umbraco.Core.VersionableRepository.GetSortOrder";
+            public const string VersionableRepositoryGetParentNode = "Umbraco.Core.VersionableRepository.GetParentNode";
+            public const string VersionableRepositoryGetReservedId = "Umbraco.Core.VersionableRepository.GetReservedId";
+
+        }
+    }
+}
```

Comment:
@Shazwazza would it be nicer to have a class `VersionableRepository`:

```
public static class SqlTemplates
{
    public static class VersionableRepository
    {
        public const string GetVersionIds = "Umbraco.Core.VersionableRepository.GetVersionIds";
        public const string GetVersion = "Umbraco.Core.VersionableRepository.GetVersion";
        public const string GetVersions = "Umbraco.Core.VersionableRepository.GetVersions";
        public const string EnsureUniqueNodeName = "Umbraco.Core.VersionableRepository.EnsureUniqueNodeName";
        public const string GetSortOrder = "Umbraco.Core.VersionableRepository.GetSortOrder";
        public const string GetParentNode = "Umbraco.Core.VersionableRepository.GetParentNode";
        public const string GetReservedId = "Umbraco.Core.VersionableRepository.GetReservedId";
    }
}
```

so when used it would be e.g. `Constants.SqlTemplates.VersionableRepository.GetVersionIds`?

Reply:
Maybe it need to reference to other things in future besides `VersionableRepository`.

Reply:
Sounds good, i'll updtae